LoRaEngine/modules/LoRaWanNetworkSrvModule/module.json

```diff
@@ -4,12 +4,11 @@
   "image": {
     "repository": "$CONTAINER_REGISTRY_ADDRESS/lorawannetworksrvmodule",
     "tag": {
-      "version": "0.1.1",
+      "version": "0.2.0",
```

Comment:
Shouldn't we try to keep that one on the same version as overall release, i.e. "0.1.0"

Reply:
problem is that I have used the 0.1.xxx so starting from a lower number of what I have in the gateways and registry needs clean up. I found it easy to have 0.2.xxx

Reply:
`LoraKeysManagerFacade/.vscode` I understand, are the others needed as well?

Reply:
fixed .vscode